src/hooks/useScreenWrapperTransitionStatus.ts

```diff
@@ -15,3 +15,9 @@ export default function useScreenWrapperTranstionStatus() {
 
     return value;
 }
+
+function useScreenWrapperTransitionStatusUnsafe() {
```

Comment:
can you explain this with a comment? Why do we need it, why is it unsafe, when is it ok to use?

Reply:
I looked once again at it and I think it's not a good idea to add this hook as it's very confusing. It's used only in one place - `useSyncFocus` hook. I'll `useContext` directly there and add comment with explanation